composer/trainer/trainer.py

```diff
@@ -978,7 +978,7 @@ def __init__(
         if eval_dataloader is None:
             evaluators: List[Evaluator] = []
         else:
-            eval_metrics = self.state.model.get_metrics(is_train=False)
+            eval_metrics = deepcopy(self.state.model.get_metrics(is_train=False))
```

Comment:
Doing this would make it so that the metrics on the model aren't updated. is this intended?

Reply:
Yes, in the case of multiple metrics, since the model metrics only returns `Accuracy()`, we need to duplicate them.